src/index.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+
 import './index.css';
-import App from './App';
+import { App } from './screens';
```

Comment:
Where you from importing App, why ? And why you change directory in this ticket ?